netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
-        })
+
+        # Lookup for webos3
+        webos_entry = self.find_by_device_description(
+            {
+                "deviceType": "urn:schemas-upnp-org:device:MediaRenderer:1",
+                "manufacturer": "LG Electronics.",
+                "modelName": "LG TV"
+            }
+        )
+
+        if not len(webos_entry):
```

Comment:
Can u advise for a better approach since I didn't knew in python how to filter if that function returns a result ?
From my tests it returns an array (or list as its called in python)
And thats how I found to test for empty arrays...or would u prefer just to create a webos3.py file in the discoverables with just the actual filter and leave this file untouched ?
PS: I'm not a python developer